src/main/java/Calculator.java

```diff
@@ -0,0 +1,20 @@
+public class Calculator {
+
+    public double add(double a1, double a2){
+        return a1 + a2;
+    }
+
+    public double subtract(double a1, double a2){
+        return a1 - a2;
+    }
+
+    public double multiply(double a1, double a2){
+        return a1 * a2;
+    }
+
+    public double divide(double a1, double a2) throws IllegalArgumentException{
+        if (a2 == 0) throw new IllegalArgumentException("The divisor cannot be equal to 0");
+        return a1 / a2;
+    }
+
+}
```

Comment:
POM.xml:

<groupId>com.calculator</groupId>
not Ridera/Task/example etc, or company name if it's commercial product, or if it's your personal project, just package structure

I put blank line betwen <properties> and <dependencies> so we could seprate it and increase readability

Also, in pom.xml and in java class as well, before </..> closing tag or before closing brace } we DO  NOT put empty line.

There is exception in java class, we CAN put empty line before CLASS closing brace:
`
    public double divide(double a1, double a2) throws IllegalArgumentException{
        if (a2 == 0) throw new IllegalArgumentException("The divisor cannot be equal to 0");
        return a1 / a2;
    }

}`
In this case it's ok


`<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>com.calculator</groupId>
    <artifactId>calculator</artifactId>
    <version>1.0-SNAPSHOT</version>

    <properties>
        <maven.compiler.source>21</maven.compiler.source>
        <maven.compiler.target>21</maven.compiler.target>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    </properties>

    <dependencies>
        <dependency>
            <groupId>org.junit.jupiter</groupId>
            <artifactId>junit-jupiter-engine</artifactId>
            <version>5.11.0</version>
            <scope>test</scope>
        </dependency>
    </dependencies>
</project>`